src/Database/Type.php

```diff
@@ -21,7 +21,7 @@
  * Encapsulates all conversion functions for values coming from database into PHP and
  * going from PHP into database.
  */
-class Type
+class Type implements TypeInterface
```

Comment:
> Would that mean that a custom class both extends and implements? 

@davidyell No, custom class would just implement the interface. But custom classes extending `Type` would also work maintaining BC.